### ISSUE TITLE
Add support for inline connectors (IDC)

### ISSUE THREAD

**Issue by formatc1702** [formatc1702/WireViz#25]
* Possibly join two logical wires into one physical wire, add up length for BOM creation
* Designators like W1_1, W1_2 or similar to group them?

**Comment by spanner888** [comment]
+1, but in more general inline connector sense. Ie allow connector to connector connection without any wire. 

This occurs when joining cables, but also at equipment front panels (in those situations where you also want to document external & internaly)

**Comment by formatc1702** [comment]
@spanner888 I think what you mention is already being covered in #134.

**Comment by spanner888** [comment]
@formatc1702 You are correct. I did try several searches and even browsed.... but did not find that one.
